packages/@okta/vuepress-site/docs/guides/set-up-self-service-registration/configure-self-service-registration-policy/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Enable and configure a self-service registration policy
+---
+
+Use a self-service registration policy to let users self register for application access. When you create the policy, you can choose the fields that you want to include on the Create Account registration form, specify how those fields are ordered, and mark which of the fields are required.
+
+When you enable a self-service registration policy, Okta enforces uniqueness for all primary email addresses. When registering, users need to provide their email address. Okta automatically uses the email address for the username and primary email address.
+
+> **Note:** The self-service registration password policy doesn't support "does not contain first name" and "does not contain last name" complexity requirements.
+
+1. In the Admin Console, go to **Directory** and then **Self-Service Registration**.
```

Comment:
Only available in the "Classic" view

Reply:
> Only available in the "Classic" view

The Admin Console is what we call the "Classic View". The Dev Console is what we call the Developer console. I can put "Classic UI" in parens if you think it'll help with the confusion.